assets/javascripts/discourse/components/modal/activity-pub-topic-info.hbs

```diff
@@ -0,0 +1,23 @@
+<DModal
```

Comment:
Let's move this to `.gjs`.

Reply:
I've converted all of the new modals to .gjs